docs/operating-scylla/admin.rst

```diff
@@ -73,7 +73,7 @@ The following addresses can be configured in scylla.yaml:
    * - broadcast_rpc_address
      - Address that is broadcasted to tell the clients to connect to. Related to rpc_address.
    * - seeds
-     - The broadcast_addresses of the existing nodes. You must specify the address of at least one existing node.
+     - The broadcast_address of an existing node. Typically, the first node in the cluster.
```

Comment:
It's still legal to specify more than one seed, no? That's useful for automation.

If anything, this should be updated to explain that it's used only to establish initial connectivity.

Reply:
> It's still legal to specify more than one seed, no? That's useful for automation.

it is still legal

> If anything, this should be updated to explain that it's used only to establish initial connectivity.

Can do.
We already have:
```
docs/getting-started/install-scylla/install-on-linux.rst:167:   * ``seeds`` - The IP address of the first node. Other nodes will use it as the first contact 
```
```
docs/kb/seed-nodes.rst:18:A Scylla seed node is a node specified with the ``seeds`` configuration parameter in ``scylla.yaml``. It is used by new node joining as the first contact point.
```
```
docs/operating-scylla/procedures/cluster-management/add-node-to-cluster.rst:60:    * **seeds** - Specifies the IP address of an existing node in the cluster. The new node will use this IP to connect to the cluster and learn the cluster topology and state.
```

Reply:
So why change the documentation to a single node?